paper/paper.md

```diff
@@ -31,54 +31,54 @@ bibliography: paper.bib
 Such changes in the stability of the fluid flow can be studied by performing numerical simulations with a model and observing its transient behavior after a certain time.
 This is, however, computationally very expensive, and in many cases infeasible.
-Instead, so called continuation methods exist, that are able to trace stable and unstable steady states in parameter space, without having to perform the expensive transient simulations [@dijkstra:05].
+Instead, so called continuation methods are able to trace stable and unstable steady states in parameter space, obviating expensive transient simulations [@dijkstra:05].
 The `TransiFlow` Python package implements a continuation framework in which fluid flow problems can be studied with the help of several computational back-ends that can, based on the needs of the user, easily be switched between.
 
 One motivation behind `TransiFlow` is that writing research software that works efficiently on a parallel computer is a challenging task.
-Therefore, numerical models are often developed as a sequential code with parallelization as an afterthought, which makes them very difficult to parallelize, or as a parallel code from the start, which makes them complicated to work with for researchers .
+Therefore, numerical models are often developed as a sequential code with parallelization as an afterthought, which makes them very difficult to parallelize, or as a parallel code from the start, which makes them complicated to work with for researchers.
 This is especially problematic since people that work with these codes generally only work with them for the duration of their project.
 If there is insufficient continuity between the projects, knowledge of how to use the codes or work on them may get lost, which renders the developed software useless.
 
-In climate modelling, this is a prominent issue, since the models are complex, are usually intercoupled with other models (e.g. ocean, atmosphere, ice), take a very long time to run (i.e. multiple months) and require large amounts of parallelism to reach a sufficient resolution (i.e. using thousands of cores for a single run) [@thies:09; @mulder:21].
+In climate modelling, this is a prominent issue, since the models are complex, are usually intercoupled with other models (e.g., ocean, atmosphere, ice), take a very long time to run (i.e., multiple months) and require large amounts of parallelism to reach a sufficient resolution (i.e., using thousands of cores for a single run) [@thies:09; @mulder:21].
 Therefore, ease of developing and using the parallel software is crucial.
 
-By abstracting away the computational back-end from the user, the user can develop their model on their own machine (e.g. a laptop) in Python using the SciPy back-end, and once the model works, run a large scale simulation on a supercomputer e.g. using the `Trilinos` back-end, which can use a combination of OpenMP, MPI and potentially GPUs, without requiring any changes to the code.
```

Comment:
"the user can develop their model" -- what does this mean? what are the building blocks that transiflow provides for developing models that are not already included?